packages/core/src/components/DismissableBanner/DismissableBanner.stories.tsx

```diff
@@ -17,50 +17,84 @@
 import React from 'react';
 import DismissableBanner from './DismissableBanner';
 import { Link, Typography } from '@material-ui/core';
+import {
+  CreateStorageApiOptions,
+  StorageApi,
+  ErrorApi,
+} from '../../../../core-api/src/apis/definitions';
```

Comment:
I think you can change these paths to import from `@backstage/core` everywhere in this PR too.

Reply:
```tsx
import { WebStorage, StorageApi, storageApiRef} from '@backstage/core'
```

should work everywhere I think

Reply:
Awesome, made those changes and all is green :)